jupyter-js-widgets/src/widget_int.js

```diff
@@ -200,23 +203,43 @@ var IntSliderView = widget.DOMWidgetView.extend({
             var readout = this.model.get('readout');
             if (readout) {
                 this.readout.style.display = '';
+                window.setTimeout(function() {
```

Comment:
oh, right, you have to do this after it's been displayed, but it can be synchronous after that, I *think*.